.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
```

Comment:
Well, why do we need `ubuntu-latest` here? Isn't this running on PRs and Issues? Just asking. If this needs to stay, we'll have it here, as is.

Reply:
As far as i have read about it , it's like github actions provides some virtual environment(s) and the job(that we want the bot to perform) runs inside this virtual environment. 
So, yeah! This pretty much needs to stay.